app/controllers/posts_controller.rb

```diff
@@ -8,16 +10,28 @@ def show
   end
 
   def create
-    @post = Post.new(post_params) # TODO: author, location
-    @post.likes = 0
-    @post.author = 'garbo'
+    @post = current_user.post.new(post_params) # TODO: author, location
+    @post.author = current_user.email
```

Comment:
again, not something you need to do in this PR, but we should prob remove the author field cuz user_id takes care of that